apps/apps.go

```diff
@@ -56,14 +58,18 @@ func New(c *deis.Client, id string) (api.App, error) {
 		}
 	}
 
-	resBody, err := c.BasicRequest("POST", "/v2/apps/", body)
-
+	res, err := c.Request("POST", "/v2/apps/", body)
 	if err != nil {
 		return api.App{}, err
 	}
+	// Fix json.Decoder bug in <go1.7
```

Comment:
Maybe this should be `// TODO: Remove when json.Decoder bug is fixed (go >= 1.7)` ? It's unfortunate that this has to be repeated everywhere `Decode` is used.

Also do you have a reference for this json.Decoder issue? Just curious.